### ISSUE TITLE
ColorVariable rule and *CSS* color functions

### ISSUE THREAD

**Issue by hail2u** [brigade/scss-lint#611]
When `rgb()`, `rgba()`, `hsl()`, and `hsla()` colors assigned to a variable, scss-lint v0.42.1 warns these colors:

```css
// test.scss
$a: rgb(1, 2, 3);
$b: rgba(1, 2, 3, 0);
$c: hsl(1, 2%, 3%);
$d: hsla(1, 2%, 3%, 0);
$e: #123456;
```

```bash
$ scss-lint test.scss
test.scss:1 [W] ColorVariable: Color literals like `rgb(1, 2, 3)` should only be used in variable declarations; they should be referred to via variable everywhere else.
test.scss:2 [W] ColorVariable: Color literals like `rgba(1, 2, 3, 0)` should only be used in variable declarations; they should be referred to via variable everywhere else.
test.scss:3 [W] ColorVariable: Color literals like `hsl(1, 2%, 3%)` should only be used in variable declarations; they should be referred to via variable everywhere else.
test.scss:4 [W] ColorVariable: Color literals like `hsla(1, 2%, 3%, 0)` should only be used in variable declarations; they should be referred to via variable everywhere else.
```

These colors should be allowed.

**Comment by webgefrickel** [comment]
I can confirm this false-positive issue with 0.42.1 and am running into it as well.

**Comment by sds** [comment]
Duplicate of #609. Fixed in #612.
